src/api/cpp/cvc5.h

```diff
@@ -350,7 +352,7 @@ class Datatype;
  */
 class CVC5_EXPORT Sort
 {
-  friend class cvc5::Command;
+  friend class parser::Command;
```

Comment:
Do we still need the friend declarations for Command?

Reply:
Yes, I think that we currently still need it, but I am working on making that obsolete.